bin/install-qa-check.d/20perl-directories

```diff
@@ -0,0 +1,22 @@
+# Check for perl modules installed in invalid directories.
+
+perl_dir_check() {
+	# See, for example, bug #876757
+	local dir f=()
+	for x in usr/share/perl5 ; do
+		if [[ -d "${ED}/${x}" ]]; then
+		   f+=( ${x} )
+		fi
+	done
+	if [[ ${#f[@]} -gt 0 ]] ; then
+		eqawarn "QA Notice: This ebuild installs perl modules into invalid directories."
+		eqawarn " Maybe PERLDIR is not set accordingly in the ebuild?"
+		eqawarn
+		eqawarn $(printf '%s\n' "${f[@]}")
```

Comment:
There's `eqatag` for that kind of thing, that also aids automated processing (not that anything that part).